src/controllers/dynakube/dynakube_controller.go

```diff
@@ -157,15 +155,17 @@ func (controller *DynakubeController) reconcileDynaKube(ctx context.Context, dkS
 	}
 	dtc, upd, err := dtcReconciler.Reconcile(ctx, dkState.Instance)
 
-	dkState.Update(upd, defaultUpdateInterval, "Token conditions updated")
+	dkState.Update(upd, "Token conditions updated")
 	if dkState.Error(err) {
 		log.Error(err, "failed to check tokens")
 		return
 	}
 	dkState.ValidTokens = true
 	if !dtcReconciler.ValidTokens {
 		dkState.ValidTokens = false
-		log.Info("paas or api token not valid", "name", dkState.Instance.GetName())
+		errMsg := "paas or api token not valid"
```

Comment:
this also happens with an invalid dataingest token, so I'd just change to error message to something like `tokens not valid`